test/test_generate_pedigree.py

```diff
@@ -0,0 +1,159 @@
+"""
```

Comment:
Are these tests being run through CI? Or is that a later change once all the small WIP PRs have been merged?